dts/bindings/bluetooth/infineon,cyw43xxx-bt-hci.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2023 Cypress Semiconductor Corporation (an Infineon company) or
+# an affiliate of Cypress Semiconductor Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    CYW43xxx Connectivity that uses Zephyr's Bluetooth Host Controller Interface UART
+    driver.
+
+compatible: "infineon,cyw43xxx-bt-hci"
+
+include: base.yaml
+
+properties:
+  bt-reg-on-gpios:
+    description: |
+      Power-up/down gpio to control the internal regulators used
+      by the Bluetooth section of CYW43xx device.
+    type: phandle-array
+    required: true
```

Comment:
Could there be a case where the pin is just wired to VCC and the design is not using any gpio to control this?

Reply:
update REG_ON to be optional